MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/location/LocationCameraControllerTest.java

```diff
@@ -283,21 +283,35 @@ public void onNewLatLngValue_cameraModeNoneIgnored() {
   @Test
   public void onNewLatLngValue_focalPointIsAdjusted() {
     MapboxMap mapboxMap = mock(MapboxMap.class);
-    UiSettings uiSettings = mock(UiSettings.class);
-    when(mapboxMap.getUiSettings()).thenReturn(uiSettings);
+    Transform transform = mock(Transform.class);
+
+    final MapboxMap.OnCameraMoveListener[] listener = {null};
```

Comment:
NIT: this doesn't have to be an array, we are always assigning the first element anyway

Reply:
`listener` is accessed from inner class, this is what lint suggests.

Reply:
:+1: